src/events.js

```diff
@@ -46,7 +58,7 @@ module.exports = function(ctx) {
   };
 
   events.drawChanged = function(event) {
-    featuresChanged = event.features;
+    event.features.forEach(f => changedFeatureIds.add(f.id));
```

Comment:
@davidtheclark The one is the store is cleared much more frequently than the one in events.js. Maybe we need a better name?

`changedFeatureIds` in events is all changed feature ids since the last time a modified event was triggered.